docs/index.bs

```diff
@@ -362,7 +362,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
             1. Let the {{ExtendableMessageEvent/data}} attribute of |e| be initialized to |clonedMessage|.
             1. Let the {{ExtendableMessageEvent/origin}} attribute of |e| be initialized to the <a lt="Unicode serialization of an origin">Unicode serialization</a> of |incumbentSettings|'s [=environment settings object/origin=].
             1. If |incumbentGlobal| is a {{ServiceWorkerGlobalScope}} object, let the {{ExtendableMessageEvent/source}} attribute of |e| be initialized to a new {{ServiceWorker}} object that represents |incumbentGlobal|'s [=ServiceWorkerGlobalScope/service worker=].
-            1. Else if |incumbentGlobal| is a {{Window}} object, let the {{ExtendableMessageEvent/source}} attribute of |e| be initialized to a new {{WindowClient}} object that represents |incumbentGlobal|'s [=/browsing context=].
+            1. Else if |incumbentGlobal| is a {{Window}} object, let the {{ExtendableMessageEvent/source}} attribute of |e| be initialized to a new {{WindowClient}} object that represents |incumbentGlobal|'s [=/environment settings object=].
```

Comment:
@wanderview, thanks for bringing this discussion first of all. Actually, I intended to describe this as the created windowClient is a thing that represents a browsing context (seems like a Window is a more relevant term?) instead of an environment settings object here because an WindowClient object captures the visual properties as well as the settings object. Reviewing this step now, I agree it's not really accurate in the implementation point of view. Maybe we should invoke [Create Window Client](https://w3c.github.io/ServiceWorker/#create-window-client) algorithm with the settings object and the related states retrieved from the Window here to create a WindowClient? WDYT?